tfhe/docs/getting_started/benchmarks.md

```diff
@@ -29,13 +29,42 @@ This measures the execution time of a single binary Boolean gate.
 | MEDIUM        | 113ms | 50.2ms            |
 
 
+## Integer
+This measures the execution time for some operation sets of tfhe-rs::integer.
+
+| Operations\bit_size | `FheUint8` | `FheUint16` | `FheUint32` | `FheUint40` | `FheUint64` | `FheUint128` | `FheUint256` |
+|---------------------|------------|-------------|-------------|-------------|-------------|--------------|--------------|
+| add                 | 153 ms     | 198 ms      | 246 ms      | 293 ms      | 308 ms      | 426 ms       | 483 ms       |
+| sub                 | 153 ms     | 197 ms      | 245 ms      | 289 ms      | 312 ms      | 435 ms       | 515 ms       |
+| bitand              | 37.5 ms    | 40.3 ms     | 42.8 ms     | 43.2 ms     | 46.3 ms     | 63.0 ms      | 63.4 ms      |
+| bitor               | 37.4 ms    | 39.9 ms     | 41.1 ms     | 42.1 ms     | 45.4 ms     | 56.0 ms      | 58.1 ms      |
+| bitxor              | 36.5 ms    | 40.4 ms     | 41.7 ms     | 42.6 ms     | 45.3 ms     | 56.5 ms      | 59.6 ms      |
+| negation            | 142 ms     | 195 ms      | 242 ms      | 282 ms      | 297 ms      | 419 ms       | 498 ms       |
```

Comment:
the table looks to be folded on gitbook and it makes it a bit hard to read as the first column is not fixed